### PR TITLE
Rasa X annotations

### PR DESCRIPTION
**Pull request by melindaloubser1**

### CODE REVIEW

**Comment by akelad** [comment]
you can ignore me melinda, i'm just adding this label and re-running stuff because i want output of an NLU eval for a demo later

File: data/nlu/nlu.md [hunk 1]
```diff
@@ -72,10 +73,18 @@
 - whats my [current balance](payment_amount)
 - What's my balance?
 - How much money do I have?
-- Yeah, I like coffee 👀 But how much money do I have?
 - Yeah, I know 😀 Since it's so nice: How much money do I have?
+- what places have I spent money?
+- Wait, I'll have to check my balance first. Can you do that please?
+- Hi, I need to know how much money is in my checking account
+- Balance please
```
Comment (akelad) [inline data/nlu/nlu.md:80]:
```suggestion
- Balance please.
```

**Comment by melindaloubser1** [comment]
Sure, Im pulling these annotations as a test set

**Comment by ArjaanBuijk** [comment]
Manually added them to NLU in https://github.com/RasaHQ/financial-demo/pull/98, for rasa 2

**Pull request closed without merging**